src/main/resources/reference.conf

```diff
@@ -2,6 +2,26 @@ akka.http.server.idle-timeout = 180 s
 akka.http.server.request-timeout = 60 s
 
 resourceTypes = {
+  admin {
```

Comment:
Instead of adding a new `admin` resource type, we should just add these actions and roles to the existing `resource_type_admin` resource type. The `resource_type_admin` resource type has a resource for every other resource type in Sam, and we can use those resources to grant per-resource-type admin access